apps/demo/src/Home.tsx

```diff
@@ -200,17 +198,14 @@ function Home() {
               object storage (AWS S3, Azure Blob Storage, or OpenIO).
             </p>
             <p>
-              This demo communicates with an HSDS test server, which serves the
-              same files as the H5Grove demo above:{' '}
+              This demo communicates with an HSDS test server, which serves some
+              of the same files as the H5Grove demo above:{' '}
               <Link to="/hsds">
                 <strong>water_224.h5</strong>
               </Link>{' '}
-              (default),{' '}
-              <Link to="/hsds?file=compressed.h5">compressed.h5</Link> (note
-              that bitshuffle is not yet supported by HSDS),{' '}
               <Link to="/hsds?file=epics.h5">epics.h5</Link>,{' '}
               <Link to="/hsds?file=grove.h5">grove.h5</Link>,{' '}
-              <Link to="/hsds?file=links.h5">links.h5</Link>,{' '}
+              <Link to="/hsds?file=sample_hsds.h5">sample_hsds.h5</Link>,{' '}
               <Link to="/hsds?file=tall.h5">tall.h5</Link>.
```

Comment:
Very few of our test files can be loaded into HSDS without errors, since they are meant to test advanced HDF5 feature and edge cases. I list only those that work, and keep HSDS' test file (`tall.h5`).